swagger/src/main/java/grpcbridge/swagger/proto/ProtoVisitor.java

```diff
@@ -0,0 +1,113 @@
+package grpcbridge.swagger.proto;
```

Comment:
This and the related classes are not Swagger dependent. I'd put them into a separate package.